gc3apps/bf.uzh.ch/gcsvBulk.py

```diff
@@ -0,0 +1,216 @@
+#! /usr/bin/env python
+#
+#   gcsvBulk.py -- Front-end script for running a MATLAB function
+#   with a given csv file as input parameters. Additionally a block
+#   size for the csv file can be specified to devide the csv in blocks
+#   that should yield as input for each node. In other words, the block size
+#   determines the number of nodes to run
+#
+#   Copyright (C) 2014, 2015 GC3, University of Zurich
+#
+#   This program is free software: you can redistribute it and/or
+#   modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+
+It uses the generic `gc3libs.cmdline.SessionBasedScript` framework.
+
+See the output of ``gcsvBulk.py --help`` for program usage
+instructions.
+
+Input parameters consists of:
```

Comment:
Please do not submit unfinished docstrings. If this is not needed, just remove it.